### ISSUE TITLE
[Bug][A11Y] Badge with icon should not be hidden to Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#957]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS18.5

### App version

0.19.0

### App build number

1757669301

### App build type

alpha

### Describe the issue

Badge with "icon" type is not reached by Voice Over

### Expected Behavior

Badge with "icon" type is informative, so it should be read (like the "count" style) by giving a label to the icon image.

### Accessibility settings

Voice Over

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.19.0**
- Build number: **1758143890**
- Build tag: **8385a6b**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.19.0**
- Build number: **1758230336**
- Build tag: **d7d948f**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.19.0**
- Build number: **1758316731**
- Build tag: **d792364**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
